sdks/python/apache_beam/runners/dataflow/internal/dependency.py

```diff
@@ -474,10 +482,32 @@ def _stage_beam_sdk_tarball(sdk_remote_location, staged_path, temp_dir):
         'type of location: %s' % sdk_remote_location)
 
 
-def get_required_container_version():
+def get_default_container_image_for_current_sdk(job_type):
   """For internal use only; no backwards-compatibility guarantees.
 
-  Returns the Google Cloud Dataflow container version for remote execution.
+  Args:
+    job_type (str): BEAM job type.
+
+  Returns:
+    str: Google Cloud Dataflow container image for remote execution.
+  """
+  if job_type == 'FNAPI_BATCH' or job_type == 'FNAPI_STREAMING':
```

Comment:
Thanks for review! I had this TODO in the beginning of the module since it concerns 2 functions. I moved it closer, into one of them.